crates/next-core/src/next_client/transforms.rs

```diff
@@ -54,14 +54,12 @@ pub async fn get_next_client_transforms_rules(
     match &context_ty {
         ClientContextType::Pages { pages_dir } => {
             if !foreign_code {
-                rules.push(
-                    get_next_pages_transforms_rule(
-                        pages_dir.clone(),
-                        ExportFilter::StripDataExports,
-                        enable_mdx_rs,
-                    )
-                    .await?,
-                );
+                rules.push(get_next_pages_transforms_rule(
+                    pages_dir.clone(),
+                    ExportFilter::StripDataExports,
+                    enable_mdx_rs,
+                    vec![],
+                )?);
```

Comment:
This should also have a `ReferenceType` though. We don't want to run this for every module